ts/LinearProgress/index.tsx

```diff
@@ -5,7 +5,7 @@ import MaterialComponent from '../Base/MaterialComponent';
 
 export interface ILinearProgressProps {
   indeterminate?: boolean;
-  reversed?: boolean;
+  reverse?: boolean;
```

Comment:
i still see this as `reversed`: https://github.com/material-components/material-components-web/tree/master/packages/mdc-linear-progress#css-modifiers

Reply:
hmm yeah it seems a bit inconsistent: https://github.com/material-components/material-components-web/tree/master/packages/mdc-linear-progress#mdclinearprogress-api
Have to think about that.